### DISCUSSION TITLE
I need help using Vue as a sub app

### DISCUSSION THREAD

**Discussion by pamenary** [umijs/umi#12494] [General]
When I create a sub-app with preset, Vue, it gives this error.

I need help using Vue as a sub app.

`import {defineConfig} from "umi";

export default defineConfig({
    npmClient: 'pnpm',
    plugins: [
        '@umijs/plugins/dist/qiankun',
    ],
    presets: [require.resolve('@umijs/preset-vue')],
    polyfill: false,
    qiankun: {
        slave: {},
    },
});`

**Comment by fz6m** [comment]
Vue app don't support `qiankun` and other features, it only has minimal features.

Because `umi` is a react-based framework.
